app/controllers/spouse_auth_only_controller.rb

```diff
@@ -0,0 +1,29 @@
+class SpouseAuthOnlyController < ApplicationController
+  layout "question"
+
+  helper_method :section_title
+  helper_method :illustration_path
+
+  def show
+    token = params[:token]
+    if token
+      intake = Intake.where(spouse_auth_token: params[:token]).first
+    end
+    if !token || !intake
+      return redirect_to not_found_path
+    end
```

Comment:
A little more concise might be:

```ruby
    intake = Intake.where(spouse_auth_token: params[:token]).first
    return redirect_to not_found_path unless intake.present?
```

Reply:
i think it would have to be `params[:token].present? && intake.present?` because `params[:token]` could be `nil` in which case it would find an intake where `spouse_auth_token` is `nil`?